tina/collections/rules.ts

```diff
@@ -1,36 +1,59 @@
 import { Collection } from "tinacms";
+import { generateGuid } from "../../src/helpers/generateGuid";
 
 export const Rules: Collection = {
   label: 'Rules',
   name: 'rules',
   path: '/rules',
   format: 'md',
+  defaultItem: () => {
+    return {
+      guid: generateGuid(),
+      created: new Date().toISOString(),
```

Comment:
check if we actually use this anywhere (I think no)

Reply:
The guid or the date?

Reply:
created date sorry - i think we just use the actual git history now

Reply:
I can't tell, but it should probably still exist in the schema to fit with the rule layout until we start changing the md formats. I've created a PBI and linked it https://github.com/SSWConsulting/SSW.Rules/issues/1543.